src/test/rpc/Simulate_test.cpp

```diff
@@ -0,0 +1,819 @@
+
+            auto resp = env.rpc("json", "simulate", to_string(params));
+            BEAST_EXPECT(
+                resp[jss::result][jss::error_message] ==
+                "Invalid field 'tx_blob'.");
+        }
+        {
+            // Non-string `tx_blob`
+            Json::Value params;
+            params[jss::tx_blob] = 1.1;
+
+            auto resp = env.rpc("json", "simulate", to_string(params));
+            BEAST_EXPECT(
+                resp[jss::result][jss::error_message] ==
+                "Invalid field 'tx_blob'.");
+        }
+        {
+            // Non-object `tx_json`
+            Json::Value params = Json::objectValue;
+            params[jss::tx_json] = "";
+
+            auto resp = env.rpc("json", "simulate", to_string(params));
+            BEAST_EXPECT(
+                resp[jss::result][jss::error_message] ==
+                "Invalid field 'tx_json', not object.");
+        }
+        {
+            // Invalid transaction
+            Json::Value params = Json::objectValue;
+            Json::Value tx_json = Json::objectValue;
+            tx_json[jss::TransactionType] = jss::Payment;
+            tx_json[jss::Account] = env.master.human();
+            params[jss::tx_json] = tx_json;
+
+            auto resp = env.rpc("json", "simulate", to_string(params));
+            BEAST_EXPECT(
+                resp[jss::result][jss::error_exception] ==
+                "Field 'Destination' is required but missing.");
+        }
+        {
+            // Bad account
+            Json::Value params;
+            Json::Value tx_json = Json::objectValue;
+            tx_json[jss::TransactionType] = jss::AccountSet;
+            tx_json[jss::Account] = "badAccount";
+            params[jss::tx_json] = tx_json;
+
+            auto resp = env.rpc("json", "simulate", to_string(params));
+            BEAST_EXPECTS(
+                resp[jss::result][jss::error] == "srcActMalformed",
+                resp[jss::result][jss::error].toStyledString());
+            BEAST_EXPECT(
+                resp[jss::result][jss::error_message] ==
+                "Invalid field 'tx.Account'.");
+        }
+        {
+            // Account doesn't exist for Sequence autofill
+            Json::Value params;
+            Json::Value tx_json = Json::objectValue;
+            tx_json[jss::TransactionType] = jss::AccountSet;
+            tx_json[jss::Account] = alice.human();
+            params[jss::tx_json] = tx_json;
+
+            auto resp = env.rpc("json", "simulate", to_string(params));
+            BEAST_EXPECT(
+                resp[jss::result][jss::error_message] ==
+                "Source account not found.");
+        }
+        {
+            // Invalid Signers field
+            Json::Value params;
+            Json::Value tx_json = Json::objectValue;
+            tx_json[jss::TransactionType] = jss::AccountSet;
+            tx_json[jss::Account] = env.master.human();
+            tx_json[sfSigners.jsonName] = "1";
```

Comment:
just `tx_json[sfSigners]` will work

Reply:
Fixed in 1c0b3a6f8ca9b038035689b68971ad5e31ba30df